frontend/taipy-gui/src/components/Taipy/Dialog.tsx

```diff
@@ -110,7 +139,22 @@ const Dialog = (props: DialogProps) => {
         return {};
     }, [width, height]);
 
-    return (
+    const getAnchorEl = useCallback(() => (refId && document.querySelector(refId)) || virtualElt, [refId]);
+
+    return popup ? (
+        <Popover
+            id={id}
+            onClose={handleAction}
+            open={open === undefined ? defaultOpen === "true" || defaultOpen === true : !!open}
+            className={`${className} ${getComponentClassName(props.children)}`}
+            sx={paperProps.sx}
+            anchorEl={getAnchorEl}
+            anchorOrigin={popoverAnchor}
```

Comment:
This we will want to customize...

Reply:
not very interesting, we're positioning the popup at the center of the HTML element whose selector is passed

Reply:
Well... an objective is to use this to mimic drop-down menus...